datumaro/util/mask_tools.py

```diff
@@ -216,11 +217,14 @@ def crop_covered_segments(segments, width, height,
 
     Returns:
         A list of input segments' parts (in the same order as input):
+
             [
+
                 [[x1,y1, x2,y2 ...], ...], # input segment #0 parts
                 mask1, # input segment #1 mask (if source segment is mask)
                 [], # when source segment is too small
                 ...
+
```

Comment:
Try to add better formatting for this section

Reply:
I added a `.. code-block::` directive in this place